cluster-autoscaler/simulator/cluster_test.go

```diff
@@ -134,3 +134,19 @@ func TestFindNone(t *testing.T) {
 		time.Now())
 	assert.NoError(t, err)
 }
+
+func TestShuffleNodes(t *testing.T) {
+	nodes := []*kube_api.Node{
+		BuildTestNode("n1", 0, 0),
+		BuildTestNode("n2", 0, 0),
+		BuildTestNode("n3", 0, 0)}
+	gotPermutation := false
+	for i := 0; i < 10000; i++ {
```

Comment:
Yeah. If random is random then the chance of flakiness is (5/6)^10000. That is approximately 1e-800 %.